provider/aws/formation/resource/memcached.json.tmpl

```diff
@@ -5,11 +5,15 @@
       "Type" : "String",
       "Default": "true",
       "AllowedValues" : [ "true", "false" ]
-    },     
+    },
     "Class": {
       "Type": "String",
       "Default": "cache.t2.micro"
     },
+    "Encrypted": {
+      "Type": "String",
+      "Default": ""
+    },
```

Comment:
The actual `memcached` resource is broken because it passes the Encrypted even if the parameter doesn't exist in the template.